src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
I think that it would be a good idea to implement this in the base feature, as current TA's are expecting this feature to be there as I have already built it. (Sorry about that)
My implementation can be found in this [gist]( https://gist.github.com/mjwsteenbergen/4e29c62a63d72525317e07ab51349b21)